examples/features/nse-composition/nse-firewall/firewall.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nse-firewall-vpp
+  name: acl-filter
```

Comment:
How important is this name change?
I understand why we need separate files for passthrough, but it seems to me that nothing bad would happen if firewall app was named nse-firewall-vpp.

Reply:
ok, done